internal/mutators/mutators.go

```diff
@@ -46,7 +46,6 @@ type Mutator struct {
 // This is a special function used for when room instance data is saved
 // It handles checking for special equality cases that the normal reflect.DeepEqual() doesn't handle the way we want.
 func (m MutatorList) SkipInstanceSave(other any) bool {
-	return false
 	m2, ok := other.(MutatorList)
 	if !ok {
 		return false
```

Comment:
This may have undesired effects. I don't think it's needed, we can delete the function altogether and re-add it later if we really need it.